pilot/pkg/features/pilot.go

```diff
@@ -313,7 +313,10 @@ var (
 			"It is safe to disable it if you are quite sure you don't need this feature").Get()
 
 	InjectionWebhookConfigName = env.RegisterStringVar("INJECTION_WEBHOOK_CONFIG_NAME", "istio-sidecar-injector",
-		"Name of the mutatingwebhookconfiguration to patch, if istioctl is not used.")
+		"Name of the mutatingwebhookconfiguration to patch, if istioctl is not used.").Get()
+
+	ValidationWebhookConfigName = env.RegisterStringVar("VALIDATION_WEBHOOK_CONFIG_NAME", "istio-istio-system",
+		"Name of the validatingwebhookconfiguration to patch. Empty will skip using cluster admin to patch.").Get()
```

Comment:
i am not sure the use case why would need skip patch webhookconfiguration.
Does galley startup when it is empty?

Reply:
When using an external control plane, the webhook accesses istiod using the URL of a secure gateway and disables the patching. See https://preliminary.istio.io/latest/docs/setup/install/external-controlplane/

IIUC, it needs to disable the caBundle patching because it is only valid for in-cluster access to the istiod service. @howardjohn, does that sound right? 

This would explain why external control plane is currently broken: https://github.com/istio/istio/pull/33288.

Reply:
Could we rename the env, really confusing and looks weird

Reply:
If you use a publically trusted cert then there is no need to patch the caBundle (or have permission to do it)

Reply:
Technically whether you need to patch it is not in cluster vs out of cluster, but rather what cert is used. But it often means the same

Reply:
Yes, so I was thinking that maybe `values.global.pilotCertProvider` != `istiod` is the "do not patch caBundle" check. Does that sound right?